validation/standard/readme.md

```diff
@@ -0,0 +1,9 @@
+# Standard Rollup Blockspace Charter definition files
+This directory contains a number of TOML files which declare various parameters, addresses, contract versions and other data which define a **standard chain** in the sense of the [Standard Rollup Charter](https://gov.optimism.io/t/season-6-draft-standard-rollup-charter/8135).
+
+Where necessary, distinct declarations are made for testnets and devnets to the configuration to vary to a greater extent.
```

Comment:
This sentence is a bit confusing to me. Suggest re-wording to something like this:

Where necessary, distinct declarations are made to allow testnets and devnets to have greater flexibility in their configurations.